web/viewer.css

```diff
@@ -1103,6 +1103,14 @@ html[dir='rtl'] .verticalToolbarSeparator {
   -webkit-overflow-scrolling: touch;
 }
 
+#thumbnailView > a:active {
+  outline: 0;
+}
+
+#thumbnailView > a:focus {
```

Comment:
Out of curiosity, is there any reason why these two blocks couldn't just be combined like so:
```css
#thumbnailView > a:active,
#thumbnailView > a:focus {
  outline: 0;
}
```

Reply:
> Out of curiosity, is there a reason why these two blocks couldn't just be combined like so:
> 
> ```css
> #thumbnailView > a:active,
> #thumbnailView > a:focus {
>   outline: 0;
> }
> ```

No, there isn't, I forgot you could group selectors like that. Thanks!